lib/src/review_service/logging_review_prompter.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:logger/logger.dart';
+import 'package:review_service/src/review_service/review_prompter.dart';
+
+/// Implementation of [ReviewPrompter] that logs information.
+final class LoggingReviewPrompter implements ReviewPrompter {
+  final Logger logger;
+
+  /// <summary>
```

Comment:
Hum I'm not sure it's a valid format for dart documentation

Reply:
@Soap-141

Reply:
shit I forgot to remove the tags

Reply:
@takla21 is right